src/backend/joanie/core/models/products.py

```diff
@@ -268,6 +269,34 @@ def generate_invoice(self):
         self.save()
         return invoice
 
+    def deliver_certificate(self):
+        """Deliver a pdf certificate for the owner of order"""
+        organization = self.course.organization
+        context_query = {
+            "student": {
+                "name": self.owner.get_full_name(),
+            },
+            "course": {
+                "name": self.product.title,  # pylint: disable=no-member
+                "description": self.product.description,  # pylint: disable=no-member
```

Comment:
```suggestion
```

This field could be remove. Its length could be large and risk to break the layout. Furthermore this kind of information, from my point of view, is not relevant on a certificate. The product title is enough.